repo/fsrepo.go

```diff
@@ -67,6 +67,26 @@ type FSRepo struct {
 
 var _ Repo = (*FSRepo)(nil)
 
+// FSRepoPath is a helper for getting the path to the repodir
+func FSRepoPath() string {
+	envRepoDir := os.Getenv("FIL_PATH")
+	if envRepoDir != "" {
+		return envRepoDir
+	}
+	return "~/.filecoin"
+}
+
+// CreateRepo provides a quick shorthand for initializing and opening a repo
+func CreateRepo(repoDir string, cfg *config.Config) (*FSRepo, error) {
```

Comment:
repodir could be easier to understand if it required a non empty repodir, and then callers used a new getrepodir() function to get the repodir from option or fsrepopath.